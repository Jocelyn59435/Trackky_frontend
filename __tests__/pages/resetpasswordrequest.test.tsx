import React from 'react';
import renderer from 'react-test-renderer';
import ResetPasswordRequest from '../../pages/resetpasswordrequest';
import { MockedProvider } from '@apollo/client/testing';

it('renders reset password request page correctly', () => {
  const tree = renderer
    .create(
      <MockedProvider>
        <ResetPasswordRequest />
      </MockedProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
