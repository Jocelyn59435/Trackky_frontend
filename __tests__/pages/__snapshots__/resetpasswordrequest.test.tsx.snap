// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ResetPasswordRequest page correctly 1`] = `
Array [
  <div
    className="bg-indigo py-3 flex justify-between items-center"
  >
    <div
      className="p-2 cursor-pointer"
    >
      <a
        className="cursor-pointer"
        href="/"
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": 50,
              "overflow": "hidden",
              "position": "relative",
              "width": 180,
            }
          }
        >
          <img
            data-nimg="fixed"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              data-nimg="fixed"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Ftrackky_header.png&w=384&q=75"
              srcSet="/_next/image?url=%2Ftrackky_header.png&w=256&q=75 1x, /_next/image?url=%2Ftrackky_header.png&w=384&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </div>
      </a>
    </div>
  </div>,
  <div
    className="text-center py-5"
  >
    <span
      className=" text-indigo font-black font-sans text-3xl pl-5"
    >
      Send Reset Password Request
    </span>
  </div>,
  <div
    className="mx-auto lg:w-2/3 md:w-5/6 sm:w-11/12 p-4 "
  >
    <form
      className="space-y-4"
      onSubmit={[Function]}
    >
      <div>
        <label
          className="block"
        >
          Email:
        </label>
        <input
          className="w-full border-darkgrey border-2 rounded-lg h-10 hover:border-indigo focus:bg-transparent"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
        />
      </div>
      <div
        style={Object {}}
      >
        <input
          className="w-full h-10 rounded-md bg-indigo-400 text-grey hover:bg-indigo disabled:opacity-50 text-lg font-bold"
          disabled={true}
          type="submit"
        />
      </div>
    </form>
  </div>,
]
`;
